ci-operator/config/openshift/origin/openshift-origin-main.yaml

```diff
@@ -297,6 +297,19 @@ tests:
   steps:
     cluster_profile: aws-2
     env:
+      OPENSHIFT_INSTALL_AWS_PUBLIC_ONLY: "true"
```

Comment:
This is changing `e2e-aws-ovn-serial`, which should not be updated. Only add the env var to your new job.